.github/workflows/wheels-pure.yml

```diff
@@ -0,0 +1,144 @@
+name: Build RAPIDS project wheels
+
+on:
+  workflow_call:
+    inputs:
+      package-name:
+        required: true
+        type: string
+      package-dir:
+        required: true
+        type: string
+      python-package-cuda-suffix:
+        required: false
+        type: string
+        default: ''
+      python-package-versioneer-override:
+        required: true
+        type: string
+      python-package-build-tag:
+        required: false
+        type: string
+        default: ''
+      test-checkout-src:
+        required: false
+        type: string
+        default: 'true'
+      test-requires:
```

Comment:
Same test requires improvements as citestwheel/actions.yml comments should be applied here too.